app/controllers/admin/users_controller.rb

```diff
@@ -1,6 +1,21 @@
 module Admin
   class UsersController < Admin::BaseController
     def show
+      @user = current_user
+    end
+
+    def edit
+      @user = current_user
+    end
+
+    def update
+      @user = current_user
+      if @user.update(user_params)
+        redirect_to dashboard_path
+      else
+        flash.now[:warning] = "Please make sure fields are updated correctly"
```

Comment:
I think we might want to use `flash[:error] = @user.errors.full_messages.join(", ")` to be consistent with the non-admin user controller. This way it is also specific about its errors